packages/App/src/Registaration/LogIn.tsx

```diff
@@ -0,0 +1,215 @@
+import React, {FC, useState, useContext} from 'react';
+import {
+  View,
+  StyleSheet,
+  TextInput,
+  ImageBackground,
+  Text,
+  TouchableOpacity,
+  Image,
+} from 'react-native';
+// import AsyncStorage from '@react-native-async-storage/async-storage';
+import BG from '../../Pics/Group32.png';
+import passwordUnvisible from '../../Pics/passwordUnvisible.png';
+import passwordVisible from '../../Pics/passwordVisible.png';
+import ButtonApp from '../components/ButtonApp';
+import {Context} from '../../App';
+import {Formik} from 'formik';
+import * as yup from 'yup';
+
+const logInSchema = yup.object({
+  email: yup.string().required().email(),
+  password: yup.string().required(),
+});
+
+const LogIn: FC = ({navigation}: any) => {
+  const HTTP = useContext(Context);
+
+  const [visiblePass, setVisiblePass] = useState<boolean>(true);
+
+  const logIn = async (emailReg: string, passwordReg: string) => {
+    const response = await HTTP.auth(
+      'http://localhost:8000/auth/login',
+      emailReg,
+      passwordReg,
+    );
+    console.log(response);
+  };
+
+  const signUpPage = () => {
+    navigation.navigate('SignIn');
+  };
+
+  return (
+    <ImageBackground style={styles.backGround} source={BG}>
+      <Text style={styles.title}>Log In</Text>
+
+      <Formik
+        initialValues={{email: '', password: ''}}
```

Comment:
`initialValues` with an object literal cause all children to re-render each time. It's better to move `initialValues` declaration upper in scope